add_trailing_comma/_plugins/with.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+import ast
+import functools
+import sys
+from typing import Iterable
+
+from tokenize_rt import Offset
+from tokenize_rt import Token
+
+from add_trailing_comma._ast_helpers import ast_to_offset
+from add_trailing_comma._data import register
+from add_trailing_comma._data import State
+from add_trailing_comma._data import TokenFunc
+from add_trailing_comma._plugins.imports import _find_import as _find_cm
```

Comment:
aliasing this unrelated function seems wrong -- also plugins generally shouldn't import from other plugins.  if there's common code it should get extracted